doc/tag-api.txt

```diff
@@ -54,6 +54,9 @@ Properties	string Type [readonly]
 
 			The object path of the adapter the tag belongs to.
 
+        array{byte} Uid [readonly]
```

Comment:
This does not seem related to fixing unneeded const on return. Please split your patches per each logical change.

Also you need to write commit msg explaining more why the change is needed and what do you fix here. There are examples in the Git history.